NEWS.md

```diff
@@ -1,3 +1,11 @@
+# Changes from 0.11.0 to 0.11.1
+
+* Add JSON tags to ImageConfiguration types: https://github.com/chainguard-dev/apko/pull/933
+
+* Pass UID and GID mapping to the tarball writer: https://github.com/chainguard-dev/apko/pull/932
+
+Full diff: https://github.com/chainguard-dev/apko/compare/v0.11.0..main
```

Comment:
This is only going to be correct until the next commit. Can this be either the current head commit or 0.11.1 in preparation?

Reply:
I just went off of what the previous news entry sent but I did think about that. I could make it the tag?